cmd/yaegi/run.go

```diff
@@ -70,13 +70,12 @@ func run(arg []string) error {
 
 	if cmd != "" {
 		_, err = i.Eval(cmd)
-		showError(err)
```

Comment:
Are we moving this one, because we only want to show the error if we are in interactive mode?